src/incubator/WheelPicker/Item.tsx

```diff
@@ -30,56 +31,28 @@ export default ({
   // value,
   itemHeight,
   onSelect,
-  offset
+  activeIndex
```

Comment:
Not using to offset makes the transition of active/inactive look bad. 
I believe you'll have to use it in order to transition nicely the colors